src/k-provider/ovp/provider.js

```diff
@@ -74,11 +88,19 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
     });
   }
 
-  doRequest(loaders: Array<RequestLoader>): Promise<any> {
-    const dataLoader = new OVPDataLoaderManager(this.playerVersion, this.partnerId, this.ks, this._networkRetryConfig);
+  doRequest(loaders: Array<RequestLoader>, ks?: string): Promise<any> {
+    let theKs: string = ks || this.userKs;
+    const dataLoader = new OVPDataLoaderManager(this.playerVersion, this.partnerId, theKs, this._networkRetryConfig);
 
     return new Promise((resolve, reject) => {
+      let isKs: boolean = true;
+      if (!theKs) {
+        isKs = false;
+        theKs = '{1:result:ks}';
+        dataLoader.add(OVPSessionLoader, {widgetId: this.widgetId});
```

Comment:
need to save the ks we get from the BE as `this.anonymousKs` (same logic we have in getMediaConfig)